_episodes/02-git.md

```diff
@@ -101,7 +109,12 @@ $ git log
 ~~~
 {: .bash}
 
-You will get an output resembling the following. This is something called your git commit log. Whenever you make a version, or checkpoint, of your project, you will be able to see information about that checkpoint using the `git log` command. The cookie cutter has already made a commit and written a message for you, and that is what we see for this first commit in the log.
+You will get an output resembling the following.
+This is something called your git *commit log*.
+Whenever you make a version, or checkpoint, of your project,
+you will be able to see information about that checkpoint using the `git log` command.
+The cookie cutter has already made a commit and written a message for you,
```

Comment:
Should this be CookieCutter or `cookiecutter` rather than "cookie cutter"?

Reply:
Yes, I think so.